google/resource_google_project_test.go

```diff
@@ -24,15 +24,42 @@ var (
 	originalPolicy *cloudresourcemanager.Policy
 )
 
+// Test that a Project resource can be created without an organization
+func TestAccGoogleProject_createWithoutOrg(t *testing.T) {
```

Comment:
We run our acceptance tests using a service account (not user credentials). Unfortunately, a service account belongs to an org and therefor, cannot create a project without a parent organization.

One solution: we could add a skip if a service account is used. This means we could run it locally but they wouldn't be ran by our CI server.

@paddycarver Any other solutions for this? Running user credentials for the CI server? Not sure it is safe/worth it.

Reply:
I added logic to skip the test if a service account is used. @paddycarver if you have a better suggestion, we can fix it in a separate PR.

Reply:
I'm late to the party here--sorry, I was out of office on Thursday--but IIRC we were talking about the possibility of getting the service account our CI runs with whitelisted for creating a project without a parent organisation, though I've lost track of where that discussion landed or what its current status is. That's my preferred solution, honestly, but we can follow up on this with a more permanent fix in a separate PR.